exasol_script_languages_developer_sandbox/templates/vm_bucket_cloudformation.jinja.yaml

```diff
@@ -0,0 +1,32 @@
+Resources:
+  VMBucket:
+    Type: 'AWS::S3::Bucket'
+    Properties:
+      AccessControl: PublicRead
+    DeletionPolicy: Retain
```

Comment:
@tkilias what's your thought about this? I think it's safer here to mark the bucket with "retain" so that we can't accidentally delete the VM's

Reply:
Need to check the doc. But from first though, yeah maybe. Is only the question, if we still can rotate them.

Reply:
As discussed offline this affects the behavior when someone deletes the stack. With 
```
DeletionPolicy: Retain
```
the Bucket remains as is when a user deletes the stack.